### PR TITLE
Fix alert_manager and spec

### PR DESCRIPTION
**Pull request by jshaughn**
- updates due to live-metrics work

### CODE REVIEW

**Comment by jshaughn** [comment]
@josejulio or @israel-hdez  Here is another one.  This one actually has some code fixes as well. There is no cassette for this spec.

File: app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb [hunk 1]
```diff
@@ -105,15 +105,21 @@ def convert_to_group_conditions(miq_alert)
...
     end
 
-    def mw_messaging_topic_metrics_by_column
-      MiddlewareMessaging.live_metrics_config['middleware_messaging_jms_topic']['supported_metrics_by_column']
+    def mw_messaging_metrics_by_column
+      all = {}
+      supported_metrics = MiddlewareMessaging.live_metrics_config['supported_metrics']
+      supported_metrics.keys.each { |resource_type|
+        type_metrics = supported_metrics[resource_type]
+        all.merge!( supported_metrics[resource_type].invert ) if type_metrics
```
Comment (israel-hdez) [inline app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: Isn't a little better:?
```ruby
all = type_metrics.invert if type_metrics

Comment (jshaughn) [reply app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: If there are multiple types (and there are already 2) then we need to merge all of the mappings into one in order to return a master hash for lookup.  It's sort of clunky but trigger definition is rare.

maybe I missed your point but I think in your suggestion 'all' would get replaced for each type, and ultimately return just the last type's mappings.

Comment (israel-hdez) [reply app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: @jshaughn yes, you are right...
Perhaps, then, `all.merge!( type_metrics.invert ) if type_metrics`
but that's will just make it a little shorter. It's OK for me. I can merge as is if you don't want to change it.

Comment (jshaughn) [reply app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: No, that will blow up if there are no type_metrics, that is why there is 'if type_metrics'.

Comment (israel-hdez) [reply app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: Indeed my suggestion has the conditional. But, anyway, I'll merge.

Comment (jshaughn) [reply app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb:120]: oh, right, I'll change it in a future update.

**Pull request merged**